CHANGELOG.md

```diff
@@ -4,36 +4,6 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-<!-- EXAMPLE
-
-## [1.0.0]
-
-### Added
-
-- I've added feature XY (#1000)
-
-### Changed
-
-- I've cleaned up XY (#1000)
-
-### Deprecated
-
-- I've deprecated XY (#1000)
-
-### Removed
-
-- I've removed XY (#1000)
-
-### Fixed
-
-- I've fixed XY (#1000)
-
-### Security
-
-- I've improved the security in XY (#1000)
-
--->
-
```

Comment:
This example template shouldn't be removed.

Reply:
Sorry, removed it in error.  I've added it back